_posts/2021-05-18-Welcome-GSoC-students.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Fortran-lang welcomes new students to Google Summer of Code 2021"
+category: newsletter
+author: Milan Curcic, Ondřej Čertík, Sebastian Ehlert, Laurence Kedward, Arjen Markus, Brad Richardson
+---
+
+We're happy to announce six students that will work on Fortran projects under
+the Google Summer of Code 2021 program:
+
+* [Aman Godara](https://github.com/aman-godara) will work on strings in the
+  [Fortran Standard Library](https://github.com/fortran-lang/stdlib). Aman's
+  mentors will be [Sebastian Ehlert](https://github.com/awvwgk) and
+  [Milan Curcic](https://github.com/milancurcic).
+* [Rohit Goswami](https://github.com/haozeke) will work on the
+  [LFortran](https://lfortran.org) compiler, specifically toward the capability
+  to compile a complex physics package. Rohit's mentor will be
+  [Ondřej Čertík](https://github.com/certik).
+* [Jakub Jelínek](https://github.com/kubajj) will work on handling compiler
+  arguments in the
+  [Fortran Package Manager](https://github.com/fortran-lang/fpm). Jakub's
+  mentors will be [Laurence Kedward](https://github.com/lkedward) and
+  [Brad Richardson](https://github.com/everythingfunctional).
+* [Chetan Karwa](https://github.com/chetankarwa) will work on the linked list
+  support in the
+  [Fortran Standard Library](https://github.com/fortran-lang/stdlib). Chetan's
+  mentors will be [Arjen Markus](https://github.com/arjenmarkus) and
+  [Milan Curcic](https://github.com/milancurcic).
+* [Thirumalai Shaktivel](https://gitlab.com/Thirumalai-Shaktivel) will work
+  on the Abstract Syntax Tree generation in the [LFortran](https://lfortran.org)
+  compiler. Thirumalai's mentor will be
+  [Ondřej Čertík](https://github.com/certik).
+* [Gagandeep Singh](https://github.com/czgdp1807) will work on the support
+  of arrays and allocatables in the [LFortran](https://lfortran.org) compiler.
+  Gagandeep's mentor will be [Ondřej Čertík](https://github.com/certik).
+  Gagandeep was accepted under the NumFOCUS application to GSoC.
+
+This is the first year that Fortran-lang applied for Google Summer of Code, and
+we're beyond excited that our project was allocated this many student slots.
+Thank you, Google, for your support of Fortran and open source software!
```

Comment:
Do you want to add something like:
```suggestion
Thank you, Google, for your support of Fortran and open source software!

You can follow students progress in their weekly reports in the [Fortran Discourse](https://fortran-lang.discourse.group/) forum.
```

Reply:
Good idea, but I'd prefer to reorder it so that Thank you Google comes last:

Do you want to add something like:
```suggestion
You can follow students' progress in their weekly reports in the [Fortran Discourse](https://fortran-lang.discourse.group/) forum.

Thank you, Google, for your support of Fortran and open source software!
```

Do you agree?

Reply:
Sure, that works.